### PR TITLE
feat: add filesize util PE-1354

### PR DESCRIPTION
**Pull request by jdaev**
Adds filesize to string util that takes byte size as input and returns a string.

### CODE REVIEW

File: lib/utils/filesize.dart [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import 'dart:math';
+
+/// A method returns a human readable string representing a file _size
+String filesize(dynamic size, [int round = 2]) {
+  /** 
+   * [size] can be passed as number or as string
```
Comment (karlprieb) [inline lib/utils/filesize.dart:6]: I think you can explain better that `size` is bytes

File: lib/pages/drive_detail/drive_detail_page.dart [hunk 2]
```diff
@@ -27,6 +26,7 @@ import 'package:timeago/timeago.dart';
 import 'package:url_launcher/url_launcher.dart';
 
 import '../../../utils/app_localizations_wrapper.dart';
+import '../../../utils/filesize.dart';
```
Comment (karlprieb) [inline lib/pages/drive_detail/drive_detail_page.dart:29]: why not `package:ardrive/utils/filesize.dart`?

Comment (jdaev) [reply lib/pages/drive_detail/drive_detail_page.dart]: [Updated](https://github.com/ardriveapp/ardrive-web/pull/460/commits/b171249380d3bbadc8207664e599c9f949dfc938)

File: lib/utils/filesize.dart (hunk 1 above)
Comment (jdaev) [reply lib/utils/filesize.dart]: [Updated](https://github.com/ardriveapp/ardrive-web/pull/460/commits/b171249380d3bbadc8207664e599c9f949dfc938)

**Review by karlprieb — APPROVED** [review]

**Review by fedellen — APPROVED** [review]
🔢

**Pull request merged**